packages/client/src/declarations/index.ts

```diff
@@ -1126,9 +1159,16 @@ export async function deleteDeclarationByUser(
 }
 
 export function downloadDeclaration(
-  declaration: IDeclaration,
+  event: Event,
+  compositionId: string,
+  action: string,
   client: ApolloClient<{}>
 ): IDownloadDeclaration {
+  const declaration = makeDeclarationReadyToDownload(
```

Comment:
Refactor `makeDeclarationReadyToDownload` to be called inside the action, no point on importing separately